docs/getting_started/basic/zero-shot-prediction/zero-shot-classification.ipynb

```diff
@@ -0,0 +1,479 @@
+{
```

Comment:
<p>As a refresher, <code>dataset.apply_model()</code> uses the model for inference on the dataset, with the parameters provided when we loaded the zoo model. It creates a dataset label field named "clip_classification" and populates each sample with the output. </p><p> </p><p>You can examine the results on the&nbsp;<a href="https://beta-docs.voxel51.com/api/fiftyone.core.dataset.Dataset.html#first" target="_blank">first Sample</a>&nbsp;as follows:</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thesteve0/v51-gs-draft/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>